### PR TITLE
Add Getting Started to README

### PR DESCRIPTION
**Pull request by elie222**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,45 @@
...
+
+4. Run setup:
+
+```sh
+yarn local:setup
+```
+
+5. Run client and server:
+
+```sh
```
Comment (gilgardosh) [inline README.md:33]: For VSCode users, there's the `Run Accounter` button on the bottom bar

File: README.md [hunk 2]
```diff
@@ -1,4 +1,45 @@
...
+yarn build
+yarn client:dev
+yarn server:dev
+```
+
+6. Visit [http://localhost:3001/](http://localhost:3001/) and sign in. The credentials to log in are
+   in your `.env` file under `AUTHORIZED_USERS`.
```
Comment (gilgardosh) [inline README.md:40]: The actual value of this env variable is json object, where attributes are user names and values are  hashed passwords (with bcrypt)

**Comment by gilgardosh** [comment]
Thank you @elie222!! Let me know if you have any questions digging in / running the app

File: README.md (hunk 1 above)
Comment (elie222) [reply README.md:33]: Nice :)

File: README.md (hunk 2 above)
Comment (elie222) [reply README.md:40]: I'll update the copy to make that more clear.

**Pull request closed without merging**